### ISSUE TITLE
Frontend - 500 Internal Server Error

### ISSUE THREAD

**Issue by VNRARA** [timmo001/system-bridge#1449]
### Description

Can't get to my settings: 
![image](https://user-images.githubusercontent.com/3942672/159590907-1be92358-87dd-42b0-aef7-158dbff7c54d.png)

### What Platform / OS are you running?

Windows

### What version are you running?

v2.8.4

### Anything in the logs or a references that might be useful?

```txt
2022-03-23 00:02:03 info: EventsGateway - New event: data-graphics - graphics - findAll
2022-03-23 00:02:10 error: CpuService - Error: Cannot read property 'value' of undefined
2022-03-23 00:02:10 info: EventsGateway - New event: data-cpu - cpu - findAll
2022-03-23 00:02:15 info: Observer - Done
2022-03-23 00:02:15 info: EventsGateway - New event: data-network - network - findAll
2022-03-23 00:02:33 info: RequestInterceptor - GET: /app/settings?apiKey=33f336de-4193-4472-99d7-af8be1488d3e&apiPort=9170&wsPort=9172 - {"0":"settings"} - {"apiKey":"33f336de-4193-4472-99d7-af8be1488d3e","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:02:51 info: Observer - Run Jobs: processes - findCurrentLoad, memory - findAll, battery - findAll, display - findAll, filesystem - findSizes, graphics - findAll, cpu - findAll, network - findAll
2022-03-23 00:02:52 info: EventsGateway - New event: data-processes-findCurrentLoad - processes - findCurrentLoad
2022-03-23 00:02:53 info: EventsGateway - New event: data-memory - memory - findAll
2022-03-23 00:02:55 info: EventsGateway - New event: data-battery - battery - findAll
2022-03-23 00:02:57 info: EventsGateway - New event: data-display - display - findAll
2022-03-23 00:02:58 info: EventsGateway - New event: data-filesystem-findSizes - filesystem - findSizes
2022-03-23 00:03:02 info: EventsGateway - New event: data-graphics - graphics - findAll
2022-03-23 00:03:10 error: CpuService - Error: Cannot read property 'value' of undefined
2022-03-23 00:03:10 info: EventsGateway - New event: data-cpu - cpu - findAll
2022-03-23 00:03:14 info: Observer - Done
2022-03-23 00:03:14 info: EventsGateway - New event: data-network - network - findAll
2022-03-23 00:03:44 info: RequestInterceptor - GET: /app/settings?apiKey=&apiPort=9170&wsPort=9172 - {"0":"settings"} - {"apiKey":"","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /_next/static/css/26d83a0d9ce6e4f4.css - {"0":"/static/css/26d83a0d9ce6e4f4.css"} - {} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /_next/static/chunks/webpack-9a7950c3b4971ae6.js - {"0":"/static/chunks/webpack-9a7950c3b4971ae6.js"} - {} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /_next/static/chunks/framework-c9d48730bbe34125.js - {"0":"/static/chunks/framework-c9d48730bbe34125.js"} - {} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /_next/static/chunks/main-3ea2eb712c4da227.js - {"0":"/static/chunks/main-3ea2eb712c4da227.js"} - {} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /_next/static/chunks/pages/_app-ddc2d33cab61fb78.js - {"0":"/static/chunks/pages/_app-ddc2d33cab61fb78.js"} - {} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /_next/static/chunks/pages/_error-f7153c1399372827.js - {"0":"/static/chunks/pages/_error-f7153c1399372827.js"} - {} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /_next/static/wd2KqNBYybDo7hYFte9-_/_buildManifest.js - {"0":"/static/wd2KqNBYybDo7hYFte9-_/_buildManifest.js"} - {} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /_next/static/wd2KqNBYybDo7hYFte9-_/_ssgManifest.js - {"0":"/static/wd2KqNBYybDo7hYFte9-_/_ssgManifest.js"} - {} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /_next/static/wd2KqNBYybDo7hYFte9-_/_middlewareManifest.js - {"0":"/static/wd2KqNBYybDo7hYFte9-_/_middlewareManifest.js"} - {} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /_next/static/media/roboto-latin-400-normal.7b8d7718.woff2 - {"0":"/static/media/roboto-latin-400-normal.7b8d7718.woff2"} - {} - 200
2022-03-23 00:03:44 info: RequestInterceptor - GET: /favicon.svg - {"0":"svg"} - {} - 200
2022-03-23 00:03:46 info: RequestInterceptor - GET: /app/settings?apiKey=&apiPort=9170&wsPort=9172 - {"0":"settings"} - {"apiKey":"","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:03:48 info: RequestInterceptor - GET: /app/settings?apiKey=&apiPort=9170&wsPort=9172 - {"0":"settings"} - {"apiKey":"","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:03:51 info: Observer - Run Jobs: processes - findCurrentLoad, memory - findAll, battery - findAll, display - findAll, filesystem - findSizes, graphics - findAll, cpu - findAll, network - findAll
2022-03-23 00:03:51 info: RequestInterceptor - GET: /app/settings?apiKey=&apiPort=9170 - {"0":"settings"} - {"apiKey":"","apiPort":"9170"} - 200
2022-03-23 00:03:52 info: EventsGateway - New event: data-processes-findCurrentLoad - processes - findCurrentLoad
2022-03-23 00:03:53 info: RequestInterceptor - GET: /app/settings?apiKey=&apiPort=9170&wsPort=9172 - {"0":"settings"} - {"apiKey":"","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:03:53 info: EventsGateway - New event: data-memory - memory - findAll
2022-03-23 00:03:55 info: EventsGateway - New event: data-battery - battery - findAll
2022-03-23 00:03:57 info: EventsGateway - New event: data-display - display - findAll
2022-03-23 00:03:58 info: EventsGateway - New event: data-filesystem-findSizes - filesystem - findSizes
2022-03-23 00:04:03 info: EventsGateway - New event: data-graphics - graphics - findAll
2022-03-23 00:04:10 error: CpuService - Error: Cannot read property 'value' of undefined
2022-03-23 00:04:10 info: EventsGateway - New event: data-cpu - cpu - findAll
2022-03-23 00:04:14 info: Observer - Done
2022-03-23 00:04:14 info: EventsGateway - New event: data-network - network - findAll
2022-03-23 00:04:21 info: EventsGateway - New client registered
2022-03-23 00:04:21 info: EventsGateway - New event: exit-application
2022-03-23 00:04:21 info: Process Manager - api: Exited with code: 0 - ["C:\\Users\\LNDN\\AppData\\Local\\System Bridge\\system-bridge.exe","C:\\snapshot\\system-bridge\\dist\\components\\api\\index.js"]
2022-03-23 00:04:21 info: Process Manager - Mopping up processes..
2022-03-23 00:04:21 info: Process Manager - Closing..
2022-03-23 00:04:21 info: Process Manager - gui: Exited with code: 0 - ["C:\\Users\\LNDN\\AppData\\Local\\System Bridge\\system-bridge-gui\\system-bridge-gui.exe","--host","localhost","--api-key","33f336de-4193-4472-99d7-af8be1488d3e","--api-port","9170","--frontend-port","9170","--log-level","info","--websocket-port","9172"]
2022-03-23 00:04:21 info: Process Manager - Mopping up processes..
2022-03-23 00:04:21 info: Process Manager - Closing..
2022-03-23 00:04:26 info: System Bridge 2.8.4: C:\Users\LNDN\AppData\Local\System Bridge - C:\Users\LNDN\AppData\Local\System Bridge\system-bridge.exe - C:\Users\LNDN\AppData\Local\System Bridge - ["C:\\Users\\LNDN\\AppData\\Local\\System Bridge\\system-bridge.exe","C:\\snapshot\\system-bridge\\dist\\index.js"] -  -  -  - 
2022-03-23 00:04:26 info: Process Manager - PATH_API: C:\snapshot\system-bridge\dist\components\api\index.js
2022-03-23 00:04:26 info: Process Manager - Starting api - ["C:\\Users\\LNDN\\AppData\\Local\\System Bridge\\system-bridge.exe","C:\\snapshot\\system-bridge\\dist\\components\\api\\index.js"]
2022-03-23 00:04:27 info: Process Manager - PATH_GUI_PACKAGED: C:\Users\LNDN\AppData\Local\System Bridge\system-bridge-gui\system-bridge-gui.exe
2022-03-23 00:04:27 info: Process Manager - Starting gui - ["C:\\Users\\LNDN\\AppData\\Local\\System Bridge\\system-bridge-gui\\system-bridge-gui.exe","--host","localhost","--api-key","***","--api-port","9170","--frontend-port","9170","--log-level","info","--websocket-port","9172"]
2022-03-23 00:04:28 info: API - C:\Users\LNDN\AppData\Local\System Bridge - C:\Users\LNDN\AppData\Local\System Bridge\system-bridge.exe - C:\Users\LNDN\AppData\Local\System Bridge - ["C:\\Users\\LNDN\\AppData\\Local\\System Bridge\\system-bridge.exe","C:\\snapshot\\system-bridge\\dist\\components\\api\\index.js"] -  -  -  - 
2022-03-23 00:04:28 info: MQTT - Setup
2022-03-23 00:04:29 info: API - API started on port 9170
2022-03-23 00:04:29 info: API - Launch on startup: true - enabled - C:\Users\LNDN\AppData\Local\System Bridge\system-bridge.exe
2022-03-23 00:04:29 info: API - RTC broker created on path /rtc
2022-03-23 00:04:29 info: EventsGateway - New client registered
2022-03-23 00:04:29 info: EventsGateway - New event: listening-for-events
2022-03-23 00:04:29 info: Events - Listening
2022-03-23 00:04:29 warn: API Execption - HTTP Exception: {"status":503,"body":{},"message":"Http Exception","path":"/information"}
2022-03-23 00:04:34 info: RequestInterceptor - GET: /information - 200
2022-03-23 00:04:44 info: RequestInterceptor - GET: /app/settings?apiKey=33f336de-4193-4472-99d7-af8be1488d3e&apiPort=9170&wsPort=9172 - {"0":"settings"} - {"apiKey":"33f336de-4193-4472-99d7-af8be1488d3e","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /_next/static/css/26d83a0d9ce6e4f4.css - {"0":"/static/css/26d83a0d9ce6e4f4.css"} - {} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /_next/static/chunks/webpack-9a7950c3b4971ae6.js - {"0":"/static/chunks/webpack-9a7950c3b4971ae6.js"} - {} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /_next/static/chunks/framework-c9d48730bbe34125.js - {"0":"/static/chunks/framework-c9d48730bbe34125.js"} - {} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /_next/static/chunks/main-3ea2eb712c4da227.js - {"0":"/static/chunks/main-3ea2eb712c4da227.js"} - {} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /_next/static/chunks/pages/_app-ddc2d33cab61fb78.js - {"0":"/static/chunks/pages/_app-ddc2d33cab61fb78.js"} - {} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /_next/static/chunks/pages/_error-f7153c1399372827.js - {"0":"/static/chunks/pages/_error-f7153c1399372827.js"} - {} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /_next/static/wd2KqNBYybDo7hYFte9-_/_buildManifest.js - {"0":"/static/wd2KqNBYybDo7hYFte9-_/_buildManifest.js"} - {} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /_next/static/wd2KqNBYybDo7hYFte9-_/_ssgManifest.js - {"0":"/static/wd2KqNBYybDo7hYFte9-_/_ssgManifest.js"} - {} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /_next/static/wd2KqNBYybDo7hYFte9-_/_middlewareManifest.js - {"0":"/static/wd2KqNBYybDo7hYFte9-_/_middlewareManifest.js"} - {} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /_next/static/media/roboto-latin-400-normal.7b8d7718.woff2 - {"0":"/static/media/roboto-latin-400-normal.7b8d7718.woff2"} - {} - 200
2022-03-23 00:04:48 info: RequestInterceptor - GET: /favicon.svg - {"0":"svg"} - {} - 200
2022-03-23 00:04:50 info: EventsGateway - New client registered
2022-03-23 00:04:50 info: EventsGateway - New event: get-data
2022-03-23 00:04:50 info: EventsGateway - New event: getting-data
2022-03-23 00:04:50 info: Events - Get data
2022-03-23 00:04:50 info: Events - Get data: {"service":"battery","method":"findAll","observe":true}
2022-03-23 00:04:50 info: Events - Get data: {"service":"cpu","method":"findAll","observe":true}
2022-03-23 00:04:50 info: Events - Get data: {"service":"display","method":"findAll","observe":true}
2022-03-23 00:04:50 info: Events - Get data: {"service":"filesystem","method":"findSizes","observe":true}
2022-03-23 00:04:50 info: Events - Get data: {"service":"graphics","method":"findAll","observe":true}
2022-03-23 00:04:50 info: Events - Get data: {"service":"memory","method":"findAll","observe":true}
2022-03-23 00:04:50 info: Events - Get data: {"service":"network","method":"findAll","observe":true}
2022-03-23 00:04:50 info: Events - Get data: {"service":"os","method":"findAll","observe":false}
2022-03-23 00:04:50 info: Events - Get data: {"service":"processes","method":"findCurrentLoad","observe":true}
2022-03-23 00:04:50 info: Events - Get data: {"service":"system","method":"findAll","observe":false}
2022-03-23 00:04:53 info: EventsGateway - New event: data-processes-findCurrentLoad - processes - findCurrentLoad
2022-03-23 00:04:53 info: Observer - Add job processes - findCurrentLoad
2022-03-23 00:04:55 info: EventsGateway - New event: data-battery - battery - findAll
2022-03-23 00:04:55 info: Observer - Add job battery - findAll
2022-03-23 00:04:56 info: EventsGateway - New event: data-filesystem-findSizes - filesystem - findSizes
2022-03-23 00:04:56 info: Observer - Add job filesystem - findSizes
2022-03-23 00:04:56 info: EventsGateway - New event: data-memory - memory - findAll
2022-03-23 00:04:56 info: Observer - Add job memory - findAll
2022-03-23 00:04:57 info: EventsGateway - New event: data-display - display - findAll
2022-03-23 00:04:57 info: Observer - Add job display - findAll
2022-03-23 00:04:59 info: EventsGateway - New event: data-graphics - graphics - findAll
2022-03-23 00:04:59 info: Observer - Add job graphics - findAll
2022-03-23 00:04:59 info: EventsGateway - New event: data-os - os - findAll
2022-03-23 00:04:59 info: RequestInterceptor - GET: /app/settings?apiKey=33f336de-4193-4472-99d7-af8be1488d3e&apiPort=9170&wsPort=9172 - {"0":"settings"} - {"apiKey":"33f336de-4193-4472-99d7-af8be1488d3e","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:05:00 info: EventsGateway - New event: data-system - system - findAll
2022-03-23 00:05:01 info: EventsGateway - New event: data-network - network - findAll
2022-03-23 00:05:01 info: Observer - Add job network - findAll
2022-03-23 00:05:02 error: CpuService - Error: Cannot read property 'value' of undefined
2022-03-23 00:05:02 info: EventsGateway - New event: data-cpu - cpu - findAll
2022-03-23 00:05:02 info: Observer - Add job cpu - findAll
2022-03-23 00:05:29 info: Observer - Run Jobs: processes - findCurrentLoad, battery - findAll, filesystem - findSizes, memory - findAll, display - findAll, graphics - findAll, network - findAll, cpu - findAll
2022-03-23 00:05:30 info: EventsGateway - New event: data-processes-findCurrentLoad - processes - findCurrentLoad
2022-03-23 00:05:31 info: EventsGateway - New event: data-battery - battery - findAll
2022-03-23 00:05:33 info: EventsGateway - New event: data-filesystem-findSizes - filesystem - findSizes
2022-03-23 00:05:33 info: RequestInterceptor - GET: /app/bridges/setup?apiKey=33f336de-4193-4472-99d7-af8be1488d3e&apiPort=9170&wsPort=9172 - {"0":"bridges/setup"} - {"apiKey":"33f336de-4193-4472-99d7-af8be1488d3e","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:05:33 info: RequestInterceptor - GET: /_next/static/chunks/b2e984c5-89a338dcda363845.js - {"0":"/static/chunks/b2e984c5-89a338dcda363845.js"} - {} - 200
2022-03-23 00:05:33 info: RequestInterceptor - GET: /_next/static/chunks/643-af89396ce1ad7eb8.js - {"0":"/static/chunks/643-af89396ce1ad7eb8.js"} - {} - 200
2022-03-23 00:05:33 info: RequestInterceptor - GET: /_next/static/chunks/301-26235970d431a1a5.js - {"0":"/static/chunks/301-26235970d431a1a5.js"} - {} - 200
2022-03-23 00:05:33 info: RequestInterceptor - GET: /_next/static/chunks/313-36201f25d6b24bfb.js - {"0":"/static/chunks/313-36201f25d6b24bfb.js"} - {} - 200
2022-03-23 00:05:33 info: RequestInterceptor - GET: /_next/static/chunks/725-a8aaf9c8c96e6892.js - {"0":"/static/chunks/725-a8aaf9c8c96e6892.js"} - {} - 200
2022-03-23 00:05:33 info: RequestInterceptor - GET: /_next/static/chunks/pages/app/bridges/setup-18adaa2ac395d901.js - {"0":"/static/chunks/pages/app/bridges/setup-18adaa2ac395d901.js"} - {} - 200
2022-03-23 00:05:33 info: RequestInterceptor - GET: /bridges - 200
2022-03-23 00:05:33 info: RequestInterceptor - GET: /information - 200
2022-03-23 00:05:33 info: RequestInterceptor - GET: /settings - 200
2022-03-23 00:05:35 info: EventsGateway - New event: data-memory - memory - findAll
2022-03-23 00:05:36 info: RequestInterceptor - GET: /information - 200
2022-03-23 00:05:37 info: EventsGateway - New event: data-display - display - findAll
2022-03-23 00:05:39 info: RequestInterceptor - GET: /bridges - 200
2022-03-23 00:05:42 info: RequestInterceptor - GET: /bridges - 200
2022-03-23 00:05:42 info: EventsGateway - New event: data-graphics - graphics - findAll
2022-03-23 00:05:46 info: EventsGateway - New event: data-network - network - findAll
2022-03-23 00:05:53 error: CpuService - Error: Cannot read property 'value' of undefined
2022-03-23 00:05:53 info: Observer - Done
2022-03-23 00:05:53 info: EventsGateway - New event: data-cpu - cpu - findAll
2022-03-23 00:06:29 info: Observer - Run Jobs: processes - findCurrentLoad, battery - findAll, filesystem - findSizes, memory - findAll, display - findAll, graphics - findAll, network - findAll, cpu - findAll
2022-03-23 00:06:30 info: EventsGateway - New event: data-processes-findCurrentLoad - processes - findCurrentLoad
2022-03-23 00:06:31 info: EventsGateway - New event: data-battery - battery - findAll
2022-03-23 00:06:33 info: EventsGateway - New event: data-filesystem-findSizes - filesystem - findSizes
2022-03-23 00:06:35 info: EventsGateway - New event: data-memory - memory - findAll
2022-03-23 00:06:36 info: RequestInterceptor - GET: /information - 200
2022-03-23 00:06:38 info: EventsGateway - New event: data-display - display - findAll
2022-03-23 00:06:39 info: RequestInterceptor - PUT: /bridges/0402826a-29ca-41b2-aca0-b174f48ca227 - {"key":"0402826a-29ca-41b2-aca0-b174f48ca227"} - {} - 200
2022-03-23 00:06:39 info: RequestInterceptor - GET: /bridges - 200
2022-03-23 00:06:42 info: EventsGateway - New event: data-graphics - graphics - findAll
2022-03-23 00:06:43 info: RequestInterceptor - GET: /bridges - 200
2022-03-23 00:06:44 info: RequestInterceptor - GET: /bridges - 200
2022-03-23 00:06:47 info: EventsGateway - New event: data-network - network - findAll
2022-03-23 00:06:50 info: RequestInterceptor - GET: /app/data?apiKey=33f336de-4193-4472-99d7-af8be1488d3e&apiPort=9170&wsPort=9172 - {"0":"data"} - {"apiKey":"33f336de-4193-4472-99d7-af8be1488d3e","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:06:50 info: RequestInterceptor - GET: /_next/static/chunks/418-5bbf5d1c4ba14276.js - {"0":"/static/chunks/418-5bbf5d1c4ba14276.js"} - {} - 200
2022-03-23 00:06:50 info: RequestInterceptor - GET: /_next/static/chunks/pages/app/data-bbc6d721b9ffbf49.js - {"0":"/static/chunks/pages/app/data-bbc6d721b9ffbf49.js"} - {} - 200
2022-03-23 00:06:51 info: RequestInterceptor - GET: /information - 200
2022-03-23 00:06:51 info: RequestInterceptor - GET: /settings - 200
2022-03-23 00:06:51 info: EventsGateway - New client registered
2022-03-23 00:06:51 info: RequestInterceptor - GET: /_next/data/wd2KqNBYybDo7hYFte9-_/app/data.json - {"0":"/data/wd2KqNBYybDo7hYFte9-_/app/data.json"} - {} - 200
2022-03-23 00:06:51 info: RequestInterceptor - GET: /_next/data/wd2KqNBYybDo7hYFte9-_/app/settings.json - {"0":"/data/wd2KqNBYybDo7hYFte9-_/app/settings.json"} - {} - 200
2022-03-23 00:06:51 info: RequestInterceptor - GET: /_next/static/chunks/529-49aa8bd01b574755.js - {"0":"/static/chunks/529-49aa8bd01b574755.js"} - {} - 200
2022-03-23 00:06:51 info: RequestInterceptor - GET: /_next/static/chunks/pages/app/settings-96c264f5ee0bd630.js - {"0":"/static/chunks/pages/app/settings-96c264f5ee0bd630.js"} - {} - 200
2022-03-23 00:06:51 info: RequestInterceptor - GET: /_next/data/wd2KqNBYybDo7hYFte9-_/app/logs.json - {"0":"/data/wd2KqNBYybDo7hYFte9-_/app/logs.json"} - {} - 200
2022-03-23 00:06:51 info: EventsGateway - New event: get-data
2022-03-23 00:06:51 info: RequestInterceptor - GET: /_next/static/chunks/pages/app/logs-3ba08c6911779ae8.js - {"0":"/static/chunks/pages/app/logs-3ba08c6911779ae8.js"} - {} - 200
2022-03-23 00:06:51 info: EventsGateway - New event: getting-data
2022-03-23 00:06:51 info: Events - Get data
2022-03-23 00:06:51 info: Events - Get data: {"service":"audio","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"battery","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"bluetooth","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"cpu","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"display","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"filesystem","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"graphics","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"information","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"memory","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"network","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"os","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"processes","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"system","method":"findAll","observe":false}
2022-03-23 00:06:51 info: Events - Get data: {"service":"usb","method":"findAll","observe":false}
2022-03-23 00:06:52 info: RequestInterceptor - GET: /_next/data/wd2KqNBYybDo7hYFte9-_/app/settings.json - {"0":"/data/wd2KqNBYybDo7hYFte9-_/app/settings.json"} - {} - 200
2022-03-23 00:06:53 info: RequestInterceptor - GET: /_next/data/wd2KqNBYybDo7hYFte9-_/app/settings.json - {"0":"/data/wd2KqNBYybDo7hYFte9-_/app/settings.json"} - {} - 200
2022-03-23 00:06:54 info: EventsGateway - New client registered
2022-03-23 00:06:54 info: RequestInterceptor - GET: /_next/data/wd2KqNBYybDo7hYFte9-_/app/settings.json - {"0":"/data/wd2KqNBYybDo7hYFte9-_/app/settings.json"} - {} - 200
2022-03-23 00:06:54 info: EventsGateway - New event: get-data
2022-03-23 00:06:54 info: EventsGateway - New event: getting-data
2022-03-23 00:06:54 info: Events - Get data
2022-03-23 00:06:54 info: Events - Get data: {"service":"battery","method":"findAll","observe":true}
2022-03-23 00:06:54 info: Events - Get data: {"service":"cpu","method":"findAll","observe":true}
2022-03-23 00:06:54 info: Events - Get data: {"service":"display","method":"findAll","observe":true}
2022-03-23 00:06:54 info: Events - Get data: {"service":"filesystem","method":"findSizes","observe":true}
2022-03-23 00:06:54 info: Events - Get data: {"service":"graphics","method":"findAll","observe":true}
2022-03-23 00:06:54 info: Events - Get data: {"service":"memory","method":"findAll","observe":true}
2022-03-23 00:06:54 info: Events - Get data: {"service":"network","method":"findAll","observe":true}
2022-03-23 00:06:54 info: Events - Get data: {"service":"os","method":"findAll","observe":false}
2022-03-23 00:06:54 info: Events - Get data: {"service":"processes","method":"findCurrentLoad","observe":true}
2022-03-23 00:06:54 info: Events - Get data: {"service":"system","method":"findAll","observe":false}
2022-03-23 00:06:54 info: RequestInterceptor - GET: /app/settings?apiKey=33f336de-4193-4472-99d7-af8be1488d3e&apiPort=9170&wsPort=9172 - {"0":"settings"} - {"apiKey":"33f336de-4193-4472-99d7-af8be1488d3e","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:06:58 error: CpuService - Error: Cannot read property 'value' of undefined
2022-03-23 00:07:06 info: Observer - Done
2022-03-23 00:07:06 info: EventsGateway - New event: data-processes-findCurrentLoad - processes - findCurrentLoad
2022-03-23 00:07:06 info: EventsGateway - New event: data-battery - battery - findAll
2022-03-23 00:07:06 info: EventsGateway - New event: data-audio - audio - findAll
2022-03-23 00:07:06 info: EventsGateway - New event: data-memory - memory - findAll
2022-03-23 00:07:06 info: EventsGateway - New event: data-usb - usb - findAll
2022-03-23 00:07:06 info: EventsGateway - New event: data-battery - battery - findAll
2022-03-23 00:07:06 info: EventsGateway - New event: data-cpu - cpu - findAll
2022-03-23 00:07:06 info: RequestInterceptor - GET: /app/settings?apiKey=33f336de-4193-4472-99d7-af8be1488d3e&apiPort=9170&wsPort=9172 - {"0":"settings"} - {"apiKey":"33f336de-4193-4472-99d7-af8be1488d3e","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:07:06 info: EventsGateway - New event: data-filesystem-findSizes - filesystem - findSizes
2022-03-23 00:07:09 info: EventsGateway - New event: data-memory - memory - findAll
2022-03-23 00:07:09 info: EventsGateway - New event: data-graphics - graphics - findAll
2022-03-23 00:07:10 info: EventsGateway - New event: data-display - display - findAll
2022-03-23 00:07:11 info: EventsGateway - New event: data-bluetooth - bluetooth - findAll
2022-03-23 00:07:13 info: EventsGateway - New event: data-system - system - findAll
2022-03-23 00:07:13 info: EventsGateway - New event: data-os - os - findAll
2022-03-23 00:07:13 info: EventsGateway - New event: data-graphics - graphics - findAll
2022-03-23 00:07:13 info: EventsGateway - New event: data-information - information - findAll
2022-03-23 00:07:13 info: EventsGateway - New event: data-filesystem - filesystem - findAll
2022-03-23 00:07:14 info: EventsGateway - New event: data-system - system - findAll
2022-03-23 00:07:14 error: CpuService - Error: Cannot read property 'value' of undefined
2022-03-23 00:07:14 info: EventsGateway - New event: data-cpu - cpu - findAll
2022-03-23 00:07:15 info: EventsGateway - New event: data-processes - processes - findAll
2022-03-23 00:07:15 error: CpuService - Error: Cannot read property 'value' of undefined
2022-03-23 00:07:15 info: EventsGateway - New event: data-cpu - cpu - findAll
2022-03-23 00:07:16 info: EventsGateway - New event: data-network - network - findAll
2022-03-23 00:07:16 info: EventsGateway - New event: data-display - display - findAll
2022-03-23 00:07:17 info: EventsGateway - New event: data-network - network - findAll
2022-03-23 00:07:18 info: EventsGateway - New event: data-os - os - findAll
2022-03-23 00:07:29 info: Observer - Run Jobs: processes - findCurrentLoad, battery - findAll, filesystem - findSizes, memory - findAll, display - findAll, graphics - findAll, network - findAll, cpu - findAll
2022-03-23 00:07:30 info: EventsGateway - New event: data-processes-findCurrentLoad - processes - findCurrentLoad
2022-03-23 00:07:31 info: EventsGateway - New event: data-battery - battery - findAll
2022-03-23 00:07:33 info: EventsGateway - New event: data-filesystem-findSizes - filesystem - findSizes
2022-03-23 00:07:34 info: EventsGateway - New event: data-memory - memory - findAll
2022-03-23 00:07:36 info: EventsGateway - New event: data-display - display - findAll
2022-03-23 00:07:40 info: EventsGateway - New event: data-graphics - graphics - findAll
2022-03-23 00:07:45 info: EventsGateway - New event: data-network - network - findAll
2022-03-23 00:07:52 error: CpuService - Error: Cannot read property 'value' of undefined
2022-03-23 00:07:52 info: Observer - Done
2022-03-23 00:07:52 info: EventsGateway - New event: data-cpu - cpu - findAll
2022-03-23 00:08:03 info: RequestInterceptor - GET: /app/logs?apiKey=33f336de-4193-4472-99d7-af8be1488d3e&apiPort=9170&wsPort=9172 - {"0":"logs"} - {"apiKey":"33f336de-4193-4472-99d7-af8be1488d3e","apiPort":"9170","wsPort":"9172"} - 200
2022-03-23 00:08:04 info: RequestInterceptor - GET: /logs - 200
2022-03-23 00:08:04 info: RequestInterceptor - GET: /information - 200
2022-03-23 00:08:04 info: RequestInterceptor - GET: /settings - 200
2022-03-23 00:08:04 info: RequestInterceptor - GET: /_next/data/wd2KqNBYybDo7hYFte9-_/app/data.json - {"0":"/data/wd2KqNBYybDo7hYFte9-_/app/data.json"} - {} - 200
2022-03-23 00:08:04 info: RequestInterceptor - GET: /_next/data/wd2KqNBYybDo7hYFte9-_/app/settings.json - {"0":"/data/wd2KqNBYybDo7hYFte9-_/app/settings.json"} - {} - 200
2022-03-23 00:08:04 info: RequestInterceptor - GET: /_next/data/wd2KqNBYybDo7hYFte9-_/app/logs.json - {"0":"/data/wd2KqNBYybDo7hYFte9-_/app/logs.json"} - {} - 200
2022-03-23 00:08:05 info: RequestInterceptor - GET: /_next/data/wd2KqNBYybDo7hYFte9-_/app/settings.json - {"0":"/data/wd2KqNBYybDo7hYFte9-_/app/settings.json"} - {} - 200
2022-03-23 00:08:07 info: RequestInterceptor - GET: /information - 200
2022-03-23 00:08:13 info: RequestInterceptor - GET: /logs - 200
2022-03-23 00:08:13 info: RequestInterceptor - GET: /logs - 200
2022-03-23 00:08:14 info: RequestInterceptor - GET: /logs - 200
```

**Comment by julianrinaldi** [comment]
I have the same issue on Linux Mint.

**Comment by boyphongsakorn** [comment]
I have the same issue on Windows 10.

**Comment by MasonCrawford** [comment]
getting the same issue on windows 10 just installed for the 1st time today

**Comment by andrewoke** [comment]
Can confirm. With 2.8.4 I get the 500. Installing 2.8.3 over top (after a restart on windows 10) it is now working.

**Comment by timmo001** [comment]
Resolved in #1551 

The release will be as soon as I'm happy that 3.0.0 is production ready
